build/build.sh

```diff
@@ -35,6 +35,10 @@ fi
 
 export CGO_ENABLED=0
 export GOARCH="${ARCH}"
+GOOS=${GOOS:-}
```

Comment:
Even if it's just for consistency, I'd recommend

```suggestion
 export GOOS=${GOOS:-}
```

Reply:
Yes, thanks, it looks much better! 👍